README.rst

```diff
@@ -190,6 +190,23 @@ Available languages are stored in a mapping, and so best edited directly in Tuto
 
 Where the first key is the abbreviation of the language to use, "flag" is which flag icon is displayed in the user interface for choosing the language, and "name" is the displayed name for that language. The mapping above shows all of the current languages supported by Superset, but please note that different languages have different levels of completion and support at this time.
 
+Adding custom Row Level Security Filters to Superset
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you want to apply custom row level security filters to Superset, you can do so by using the
```

Comment:
I think it's worth saying why someone may need to do this.. and could you add a link to their docs somewhere?

E.g., 

```suggestion
if you add new datasources, tables, or fields to Superset, you may want to add new `row level security filters`_ to restrict access to that data based on people's course roles. To apply custom row level security filters to Superset, you can do so by using the
```

Link ref: https://superset.apache.org/docs/security/#row-level-security